### DISCUSSION TITLE
🌳XSOSI 计划 9 月份数据

### DISCUSSION THREAD

**Discussion by will-ww** [X-lab2017/open-wonderland#196] [2️⃣ 聊项目]
接 XSOSI 计划 [6 月份数据](https://github.com/orgs/X-lab2017/discussions/115)、[7 月份的数据](https://github.com/X-lab2017/open-wonderland/discussions/149)，[8 月份的数据](https://github.com/X-lab2017/open-wonderland/discussions/173)，9 月份的数据如下：

代码：https://github.com/X-lab2017/open-digger/blob/master/notebook/community_analysis/xlab.ipynb

![image](https://user-images.githubusercontent.com/15010826/194707404-9d4d85f4-96b5-4745-b63b-51dd9e53ce7f.png)
![image](https://user-images.githubusercontent.com/15010826/194707431-ebd632e4-4055-4390-8d1a-a8312ff533ff.png)

相较于 7 月份（姑且作为一个基数），总体影响力增加了 23%，部分原因应该是加入了 OpenEduTech 组织旗下的几个新创项目~

欢迎大家给出洞察，讨论，以及各种建议，咱们持续发展~

**Comment by will-ww** [comment]
想稍微再分析下，这 23% 的增长，有多少是 OpenEduTech 下面的项目带来的？@frank-zsy

目前数值是：3.17（OpenEduResearch） + 2.5（OpenEduKG）= 5.67，占总影响力 38.96 的 14.55%

感觉 X-lab 下面的项目应该还是有点小增长的，不知道对不对~

**Reply by frank-zsy** [reply]
是的，不算这两个项目有 8% 左右的增长
